website/events-listener/src/producer.ts

```diff
@@ -1,9 +1,8 @@
-import { Kafka, Producer } from 'kafkajs';
+import { logger } from '@gear-js/common';
 
-import { logger } from './logger';
+import { Kafka, Producer } from 'kafkajs';
 import config from './config';
-
-const log = logger('KafkaProducer');
+import { EVENTS_LISTENER } from './index';
```

Comment:
I suppose, there is a circular dependency here. KafkaProducer exported here is imported in index.ts and EVENTS_LISTENER exported in index is imported here.

Reply:
Moved `const` service name to `common` ----> `constant.ts` file.

![image](https://user-images.githubusercontent.com/50825098/173302903-d97d42de-9235-4e0d-a4b6-63c5181bb27a.png)


Now looks like this. What do you think  @osipov-mit ?
```javascript 
import { EVENTS_LISTENER } from "./common/constant";
```